modules/api/docs/LorisRESTAPI_v0.0.4-dev.md

```diff
@@ -823,90 +823,192 @@ The JSON object is of the form:
 
 ## 5.0 DICOM Data
 
-Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$Visit` 
-portion of the REST API namespace, but is defined in a separate section of this 
+Like the imaging data, the DICOM data mostly lives in the `/candidates/$CandID/$VisitLabel`
+portion of the REST API namespace, but is defined in a separate section of this
 document for clarity purposes.
 
 ### 5.1 Candidate DICOMs
+
 ```
-GET /candidates/$CandID/$Visit/dicoms
+GET /candidates/$CandID/$VisitLabel/dicoms
 ```
 
-A GET request to `/candidates/$CandID/$Visit/dicoms` will return a JSON object of
-all the raw DICOM data which have been acquired for that visit. It will return an 
-object of the form:
+A GET request to `/candidates/$CandID/$VisitLabel/dicoms` will return a JSON object of
+all the DICOM archives (A.K.A. one DICOM archive per `StudyInstanceUID`) which have been
```

Comment:
It's not a huge deal but I don't think the "a.k.a" parenthesis belongs in technical documentation. It sounds very informal.